bindings/rascal/representations/coulomb_matrix.py

```diff
@@ -106,6 +106,6 @@ def get_size(self, managers):
         Nneigh = []
         for manager in managers:
             for center in manager:
-                Nneigh.append(center.size+1)
+                Nneigh.append(center.nb_pairs+1)
```

Comment:
on the python side I prefer having nb_pairs, nb_triplets... rather than having the user know what an order and cluster is. What is your concern suggestion ?

Reply:
Ok, I see. My suggestion would have been:
instead of
```
center.nb_pairs
```
something like
```
center.get_nb_clusters(2)
```
As a read/write `const`/`non const` combination. But I see your point.